tensorboard/webapp/runs/views/runs_table/runs_table_container.ts

```diff
@@ -372,6 +373,10 @@ export class RunsTableContainer implements OnInit, OnDestroy {
     })
   );
 
+  columnFilters$ = this.store.select(
```

Comment:
I am a bit confused here. It looks like you are adding a new observable but, I do not see it being used anywhere.

Reply:
I seem to have included some changes in this PR that were not needed. I am reverting this.